### ISSUE TITLE
[Bug]: Device Sync has no way to remove another device or wipe the store

### ISSUE THREAD

**Issue by ibanner56** [ibanner56/CallersCompendium#1360]
### What happened?
ADR-004 and the sync spec lean on two server-side removals: `DELETE /v1/manifests/{deviceId}` to retire a device that is gone for good, and `DELETE /v1/store` to wipe everything under a sync ID. The server implements both (`server/lib/src/athenaeum_app.dart:343`, `:457`). Nothing in the app can issue either — the sync HTTP client has no DELETE method of any kind, and there is no `'DELETE'` string anywhere in `app/lib`.

Both gaps sit on the same missing surface and close with one PR: the Device Sync settings section that merged PR #1341 is giving a "Disconnect this device" action is where a device list and a wipe action belong. They are separate findings because the remedies differ — one removes a peer, the other destroys the store for everyone — and each needs its own confirmation copy.

**Findings in this issue**
- [ ] **1. No way to remove another device** — `DELETE /v1/manifests/{deviceId}` is named as *the* remedy for a dead device by ADR-004, `sync.md` and spec §3.3, and no client, tool or runbook issues it (`app/lib/src/sync/sync_http_client.dart:154-191`).
- [ ] **2. No way to wipe the store** — `DELETE /v1/store` is the ADR's owner-side remedy and, per §5.3, "the only in-band remedy that acts at once", and no client code issues it either (same absence; `docs/design/sync-spec.md:1690-1692`).

#### 1. No way to remove another device
Three documents name the endpoint as the remedy: ADR-004 (`docs/adr/004-device-sync-and-athenaeum.md:1288-1290`, "pruning depends in practice on dead devices being removed via `DELETE /v1/manifests/{deviceId}`"), `docs/design/sync.md:322-328` ("Pruning is therefore bounded in practice by someone calling…"), and spec §3.3 (`docs/design/sync-spec.md:329-331`, the identifier "lives for the life of the store, with `DELETE /v1/manifests/{deviceId}` as the remedy"). Nothing can send it: the app's HTTP client defines `createStore`, `getManifest`, `putManifest`, `getBlob`, `postMissing` and `putBlob` only (`app/lib/src/sync/sync_http_client.dart:154-191`); Settings has no device list; the server CLI's only store access is a break-glass *read* (`server/bin/athenaeum.dart`); the operations runbook has no procedure; and no ADR-004 tracking unit lists device removal. Detach deliberately leaves this device's own manifest in place (`docs/design/sync-spec.md:252-254`), which PR #1341 implements. A gone device's manifest therefore pins its id aliases on every peer indefinitely (`app/lib/src/sync/sync_coordinator.dart:1029-1036` retires an alias only once no current peer manifest lists the losing id), keeps its protocol identifier for the life of the store, holds one of 32 device slots so the 33rd device's first publish gets `507` (`server/lib/src/athenaeum_store.dart:377-394`), and keeps its bytes counted and its blobs reachable for GC (`:357-368`, `:456-460`).

#### 2. No way to wipe the store
ADR-004 (`docs/adr/004-device-sync-and-athenaeum.md:575-577`) gives wipe as one of the two paths that together cover every case: "Active stores are wiped by their owner, at any time, via wipe — which issues `DELETE /v1/store` and removes everything under the sync ID." The spec's glossary defines wipe the same way (`docs/design/sync-spec.md:45`), §5.3 calls `DELETE /v1/store` "the only in-band remedy that acts at once" (`:1690-1692`), and §7.3 makes it the one deletion that "MUST delete unconditionally", ignoring the blob grace window, because "a wiped store has no manifest coming … and breaks a promise the user was given explicitly" (`:3086-3090`). It is also the practical half of the ADR's leaked-credential remedy: a leaked sync ID has "no recovery and no revocation … the only remedy is to move to a new ID on every device" (`:954-956`), which leaves the old store readable by whoever holds the phrase unless it is wiped. The server implements the endpoint (`server/lib/src/athenaeum_app.dart:343`, `_methodNotAllowed(const ['GET', 'POST', 'DELETE'])` at `:351`); no client code issues it.

**I verified the absence in this session, at `main` @ fad9d2b6:**
```
$ grep -rn "'DELETE'\|\"DELETE\"" --include=*.dart app/lib packages/compendium_core/lib tools
(no output)   exit=1
$ grep -rn "deleteStore\|deleteManifest\|v1/store\|v1/manifests" --include=*.dart app/lib
app/lib/src/sync/sync_controller.dart:23:  /// `GET /v1/store`. Never issues a creation request (spec §6.2 step 2).
app/lib/src/sync/sync_controller.dart:27:  /// `POST /v1/store`, issued only after the user explicitly chose "create".
app/lib/src/sync/sync_controller.dart:363:      // failure (e.g. a 500 from `POST /v1/store`) leaves the coordinator's
app/lib/src/sync/sync_coordinator.dart:740:    // /v1/store` may already have created the store — but the user's decision
$ grep -n "Future<SyncHttpResponse>" app/lib/src/sync/sync_http_client.dart
154:  Future<SyncHttpResponse> createStore() => request('POST', 'store');
157:  …getManifest  167:  …putManifest  176:  …getBlob  183:  …postMissing  191:  …putBlob
```
The generic `request(` at `:203` is public, but no caller passes `'DELETE'`.

### What did you expect to happen?
Both remedies the design relies on are reachable from the app. The intended design, settled by the maintainer on 2026-09-22:

1. The same Device Sync settings surface that offers **"Disconnect this device"** (merged PR #1341, `app/lib/src/screens/settings/device_sync_section.dart`) gains a click-through listing the **other** devices in the store. Each is removable via `DELETE /v1/manifests/{deviceId}`, behind a warning dialog that says the removed device **can pair again later**.
2. That same surface also offers **"disconnect all devices and delete the store from the server"**, issuing `DELETE /v1/store`.

This builds on #1341 rather than duplicating it: detach stays a purely local forget that sends no request, and these two are the server-side actions beside it.

### Steps to reproduce
**Finding 1:**
1. Pair devices A and B to one store; both publish manifests.
2. Retire device B permanently (lost or wiped phone) without detaching.
3. On device A, look for any way to remove B: Settings ▸ Device Sync has no device list and no remove action, and no request path in `app/lib` issues `DELETE /v1/manifests/{deviceId}` (greps above).
4. B's manifest stays as long as A keeps the store alive, and so do its aliases, device slot and quota share. After 32 such replacements the next new device's first manifest `PUT` receives `507`.

**Finding 2:**
1. Pair a device, sync, then decide the phrase has leaked (or simply that the store should go).
2. Look for any way to delete the store from the server: Settings ▸ Device Sync offers only the sync toggle, status and — with PR #1341 — "Disconnect this device", which is local and sends no request.
3. The store and every blob in it remain until the 30-day disuse TTL reaps it, and anyone holding the phrase keeps full read and write access in the meantime (`docs/design/sync-spec.md:3391-3393`).

### Platform(s)
Linux, macOS, Windows, Android, iOS / iPadOS

### App version
main @ fad9d2b6 (unreleased — Device Sync is not in any tag; newest is v0.4.1-beta). The absence is unchanged at the deployed revision (`git show 9cac24dc:app/lib/src/sync/sync_http_client.dart | grep -c "'DELETE'"` → `0`). Latent in shipped release builds, which have no sync endpoint (`CALLERS_COMPENDIUM_SYNC_ENDPOINT` is a build-time define no release workflow sets, `app/lib/src/sync/sync_runtime.dart:93-95`); live in any build with that define set.

### Privacy acknowledgement
- [x] I haven't included personal or identifying information in this report.

**Comment by ibanner56** [comment]
**Finding 1 — no device removal.** ADR-004 (`docs/adr/004-device-sync-and-athenaeum.md:1288-1290`), `docs/design/sync.md:322-328` and spec §3.3 (`docs/design/sync-spec.md:329-331`) all name `DELETE /v1/manifests/{deviceId}` as how a genuinely gone device is removed. Nothing issues it: the app's HTTP client has no DELETE method (`app/lib/src/sync/sync_http_client.dart:154-191`), Settings has no device list, the server CLI offers only a break-glass read (`server/bin/athenaeum.dart`), `docs/dev/athenaeum-operations.md` has no procedure, and no tracking unit lists it (W13 produces the pairing/settings surfaces; W7 the reconciliation ones). A dead device's manifest pins its aliases on every peer (`app/lib/src/sync/sync_coordinator.dart:1029-1036`), keeps its identifier for the life of the store, holds one of 32 device slots so the 33rd publish gets `507` (`server/lib/src/athenaeum_store.dart:377-394`), and counts toward quota while keeping its blobs reachable (`:357-368`, `:456-460`).

**Finding 2 — no wipe.** The client never issues `DELETE /v1/store` either. The design treats wipe as a first-class owner remedy: ADR-004 `:575-577` ("Active stores are wiped by their owner, at any time, via wipe"), glossary `sync-spec.md:45`, §5.3 `:1690-1692` ("the only in-band remedy that acts at once", and destructive, so not the first response to a quota error), §7.3 `:3086-3090` (it "MUST delete unconditionally", grace window or not). It is also what makes the ADR's leaked-credential story true — `:954-956` says a leaked ID's "only remedy is to move to a new ID on every device", which leaves the old store readable to the leak-holder unless it is wiped, and §3.3's threat model states that anyone holding the ID has full read and write access including `DELETE /v1/store` (`:3391-3393`). The server route exists (`server/lib/src/athenaeum_app.dart:343`, `:351`).

**Absence verified in this session** at `main` @ fad9d2b6: `grep -rn "'DELETE'\|\"DELETE\"" --include=*.dart app/lib packages/compendium_core/lib tools` → no output, exit 1; `grep -rn "deleteStore\|deleteManifest\|v1/store\|v1/manifests" --include=*.dart app/lib` returns only three doc comments and one inline comment about `POST /v1/store` (`sync_controller.dart:23,27,363`, `sync_coordinator.dart:740`); the client's six request methods are `createStore`, `getManifest`, `putManifest`, `getBlob`, `postMissing`, `putBlob` (`sync_http_client.dart:154-191`). The generic `request(` at `:203` is public but no caller passes `'DELETE'`.

**Intended design (maintainer, 2026-09-22).** Both remedies land on the Device Sync settings surface that merged PR #1341 ("Let a device disconnect from its Device Sync store", branch `isaacbanner/device-sync-detach`) is already extending:
- **Build on #1341, do not duplicate it.** That PR adds `SyncController.detach()` plus, in `app/lib/src/screens/settings/device_sync_section.dart`, a `ListTile` keyed `sync-disconnect` rendered `if (controller.paired)`, disabled while `controller.running`, whose `_confirmDisconnect` shows an `AlertDialog` keyed `sync-disconnect-dialog` with cancel/confirm actions and reports failure via a snackbar. Detach is purely local, sends no request, and leaves this device's manifest and every peer untouched. The two new actions go beside it and reuse that confirm-dialog shape and the `controller.running` gating.
- **Other devices.** A click-through from that surface lists the **other** devices in the store — the peer set from `GET /v1/store`'s `devices`, excluding this device's own id (`docs/design/sync-spec.md:1557` and the note at `:1561-1562` that a client must exclude itself). Each entry is removable with `DELETE /v1/manifests/{deviceId}`, behind a warning dialog whose copy says the removed device **can pair again later** — it is not a ban, and a merely dormant device re-reconciles on its next pass (`docs/design/sync.md:329-334`).
- **The whole store.** The same surface offers "disconnect all devices and delete the store from the server", issuing `DELETE /v1/store`. Its confirmation must be the stronger of the two: irreversible, affects every device at once, and distinct from detach (`sync-spec.md:45`). Per §5.3 it must not be offered as the first response to a `507` quota error.

**Files to touch.**
- `app/lib/src/sync/sync_http_client.dart` — a `deleteManifest(String deviceId)` and a `deleteStore()`, both through the existing `request(` path.
- `app/lib/src/sync/sync_controller.dart` / `sync_coordinator.dart` — transport surface (near `sync_coordinator.dart:316`/`:348`) and controller methods, following #1341's writer-boundary handling for a destructive action that must not race a pass.
- `app/lib/src/screens/settings/device_sync_section.dart` — the device list click-through and the wipe action, beside #1341's `sync-disconnect` tile.
- `app/lib/l10n/app_en.arb` and the translated ARBs (#1341 adds six strings via `tools/ci/arb_translate.py`; follow the same route).
- `docs/user/settings.md` and `app/assets/docs/user/settings.md` — both copies, as #1341 does; the Device Sync section already covers connect/disconnect.
- Docs to update once a remedy exists: ADR-004 `:1288-1290` and `docs/design/sync.md:328` should name the in-app control instead of an unreachable endpoint; spec §3.3 `:329-331` likewise.

**Tests.** Client tests that removal issues exactly one `DELETE /v1/manifests/{deviceId}` for the chosen peer and never for this device's own id; that the device list excludes self; that wipe issues exactly one `DELETE /v1/store`. Widget tests mirroring #1341's (`app/test/screens/settings_screen_test.dart`): the actions appear only while paired, are disabled during a pass, cancel changes nothing, and confirm calls through. Server-side tests for `DELETE /v1/manifests/{deviceId}` are not covered today (`grep -n "DELETE" server/test/athenaeum_app_test.dart | grep -i manifest` → no output) and should be added with the status-code work tracked separately.

**Acceptance criteria**
1. A user can open the device list from Settings ▸ Device Sync, see the other devices in the store (never their own), and remove one behind a warning dialog that says it can pair again later. The removed id disappears from `GET /v1/store`'s `devices`; aliases held only by that peer retire on the next pass; a 33rd device can attach after one removal.
2. A user can delete the whole store from the same surface behind its own stronger confirmation. Afterwards `GET /v1/store` returns `404` on every device, and each device follows the §6.3 missing-store flow rather than presenting a failure.
3. Neither action is offered while unpaired or mid-pass, matching #1341's gating. Detach's behaviour is unchanged: still local, still sends no request.

**Known constraint.** `GET /v1/store` returns only opaque device ids and does not expose `written_at`, so the list cannot show "last seen" without an API change. Decide whether to ship the list with ids alone (plus "this device" suppressed) or to extend the endpoint; the removal warning carries the risk either way, since the copy states the device can pair again.

**Rejected alternatives** (from N1's escalation, now settled by the decision above): an operator/support tool — a new audited server CLI like `--break-glass` that deletes a manifest given a sync ID the user supplies — was rejected because it makes the user hand over the one bearer secret the design says never leaves them. Documenting the limitation (`type: docs`, rewriting ADR `:1288-1290`, `sync.md:328` and spec `:329-331` to say no remover exists) was rejected because it leaves a user at the 32-device cap with no in-band recovery and makes §3.3's disclosed remedy false.

**Comment by ibanner56** [comment]
**Correction: the "latent in release builds" line in this issue is stale.**

This issue's analysis says the sync paths are unreachable in shipped builds because no endpoint is configured (`CALLERS_COMPENDIUM_SYNC_ENDPOINT`). That was true when the issue was written and stopped being true when #1339 merged (3c77e186): the Athenaeum endpoint is now a stored `sync_endpoint` setting written at pairing (`app/lib/src/sync/sync_runtime.dart:47-53`), the build-time define is gone from the tree entirely, and `DeviceSyncSection` mounts unconditionally (`app/lib/src/screens/settings/experimental_section.dart:29`).

So any sync defect described here is **live for any user who has paired this device to a store**, not gated by build flavour. Nothing else in the analysis changes: the mechanism, the file:line evidence and the acceptance criteria all still hold, and no severity label is being changed on the strength of this note alone.

Raised by the #1354 fix session and verified independently against `main`.
